.gitpod.yml

```diff
@@ -6,19 +6,40 @@ github:
     branches: true
     # enable for pull requests coming from this repo (defaults to true)
     pullRequests: true
+    # enable for pull requests coming from forks (defaults to false)
+    pullRequestsFromForks: true
     # add a "Review in Gitpod" button as a comment to pull requests (defaults to false)
     addComment: true
     # configure whether Gitpod registers itself as a status check to pull requests
     addCheck: false
 
 tasks:
   - name: prebuild
-    init: ./scripts/start.sh && ./scripts/stop.sh && gp sync-done prebuild
-    command: curl -L --http1.1 https://cnfl.io/ccloud-cli | sudo sh -s -- -b /usr/local/bin; exit
+    init: |
+      cd .. && git clone https://github.com/confluentinc/jmx-monitoring-stacks && cd cp-demo/. && \
```

Comment:
I think the jmx repo should only be cloned if the user specifies the MONITORING_STACK variable. I think this should move to the "cp-demo" task